articles/api/authentication/_logout.md

```diff
@@ -24,50 +32,62 @@ curl --request GET \
 
 
 ### Request Parameters
 
-| Parameter        | Description |
-|:-----------------|:------------|
-| `returnTo `      | URL to redirect the user after the logout. |
-| `client_id`      | The `client_id` of your application. |
-| `federated`      | Add this query string parameter to the logout URL, to log the user out of their identity provider, as well: `https://${account.namespace}/v2/logout?federated`. |
+| Parameter                             | Description                                                                                                                                                     |
+| :------------------------------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id_token_hint` (Recommended)         | Previously issued ID Token for the user. This is used to indicate which user to log out                                                                         |
+| `logout_hint` (Optional)              | Optional `sub` (user ID) or `sid` (session ID) value to indicate which user to log out.                                                                         |
+| `post_logout_redirect_uri` (Optional) | URL to redirect the user after the logout.                                                                                                                      |
+| `client_id` (Optional)                | The `client_id` of your application.                                                                                                                            |
+| `federated` (Optional)                | Add this query string parameter to the logout URL, to log the user out of their identity provider, as well: `https://YOUR_DOMAIN/oidc/logout?federated`.        |
+| `state` (Optional)                    | An opaque value the applications adds to the initial request that the authorization server includes when redirecting the back to the`post_logout_redirect_uri`. |
+| `ui_locales` (Optional)               | Space-delimited list of locales used to constrain the language list for the request. The first locale on the list must match the enabled locale in your tenant  |
 
 
 ### Test with Authentication API Debugger
 
 <%= include('../../_includes/_test-this-endpoint') %>
 
-1. At the *Configuration* tab, set the fields **Application** (select the application you want to use for the test) and **Connection** (the name of the social connection to use).
+1. At the Configuration tab, set the fields **Application** (select the application you want to use for the test) and **Connection** (the name of the social connection to use).
```

Comment:
*Configuration* should be italicized to be consistent with the rest of the API docs: https://auth0.com/docs/api/authentication#idp-initiated-single-sign-on-sso-flow